### ISSUE TITLE
 Cannot read property 'RNFSFileTypeRegular' of undefined

### ISSUE THREAD

**Issue by chiuchiu328** [itinance/react-native-fs#668]
I follow the readme by first npm install and react-native link
first I go the message that I react-native run-ios
 **Cannot read property 'RNFSFileTypeRegular' of undefined**

After reading the past issue first I try
1. re-run react-native run-ios -> the same
2. delete node_module && npm install -> the same
3. open the xcodeproject  try to manually link, find that react-native link did not work. Also follow by the readme manually link. -> the same 
And another error occur, I run in xcode I got 

**clang: error: no such file or directory:** '/Users/user/Library/Developer/Xcode/DerivedData/projectName/ewngyitsgdvemmbmfxyyzhamybsh/Build/Products/Debug-appletvos/**libRNFS.a**'
**Command Ld failed with a nonzero exit code**

My environment:
"react": "16.8.3",
"react-native": "^0.58.6",
"react-native-fs": "^2.13.3"
gradle:4.10.23
macOS:macOS 10.14.4 (18E226)
Xcode: 10.2.1

please help, many thanks

**Comment by anyweez** [comment]
I ran into this issue as well; there are a number of useful tips to try here:
https://github.com/itinance/react-native-fs/issues/190

**Comment by chiuchiu328** [comment]
> I ran into this issue as well; there are a number of useful tips to try here:
> #190

thanks, I had tried those before, but all of them did not work.
I solved my problem my manual link, but there are slight different with the document
The first step and third step are the same. In the second step, I just link the first one libRNFS.a which is for iphone. Since I ran into a problem which a package can not link tvOS correctly. The solution is that disable the tvOS in the project target. So I think the problem is here. 
1. In XCode, in the project navigator, right click Libraries ➜ Add Files to [your project's name] Go to node_modules ➜ react-native-fs and add the .xcodeproj file

2. In XCode, in the project navigator, select your project. 
Add the **lib\*.a** (**here I link the one for iphone, tvOS did not link**)  from the RNFS project to your project's Build Phases ➜ Link Binary With Libraries. 

3. Click the .xcodeproj file you added before in the project navigator and go the Build Settings tab. Make sure 'All' is toggled on (instead of 'Basic'). Look for Header Search Paths and make sure it contains both $(SRCROOT)/../react-native/React and $(SRCROOT)/../../React - mark both as recursive.

Run your project (Cmd+R)
